migrations/0002_06.08.2021-migrations.up.sql

```diff
@@ -0,0 +1,61 @@
+CREATE TABLE IF NOT EXISTS roles (
+   id serial PRIMARY KEY,
+   name varchar NOT NULL,
+   description varchar NOT NULL
+);
+
+CREATE TABLE IF NOT EXISTS permisions (
+   id serial PRIMARY KEY,
+   name varchar NOT NULL,
+   description varchar NOT NULL,
+   role_id int REFERENCES roles (id)
+);
+
+CREATE TABLE IF NOT EXISTS users (
+   id uuid PRIMARY KEY DEFAULT uuid_generate_v4 (),
+   name varchar NOT NULL,
+   email varchar NOT NULL,
+   password_hash varchar NOT NULL,
+   role_id int REFERENCES roles (id)
+);
+
+CREATE TABLE IF NOT EXISTS cards (
+   id uuid PRIMARY KEY
+);
+
+CREATE TABLE IF NOT EXISTS patients (
+   id uuid PRIMARY KEY,
+   full_name varchar(50),
```

Comment:
if users have a name, why is there "full name" and not "second name"

Reply:
the user name is not a first name or last name,  I will change it to first and last names